src/main/java/client/StoreClient.java

```diff
@@ -1,116 +1,149 @@
+                .statusCode(HttpStatus.SC_OK)
+                .spec(buildUncheckedRes())
+                .log()
+                .body()
+                .extract()
+                .response()
+                .as(ResponseInfo.class);
     }
 
     @Step("Sending GET request for non existent order to /store/order by ID {0}")
     public static ResponseInfo getNonexistentOrderById(long orderId) {
         LOGGER.debug("sending request");
         RequestSpecification res = RestAssured.given()
-            .spec(buildReq());
+                .spec(buildReq());
 
         LOGGER.debug("expecting response");
         return res
-            .when()
-            .get(String.valueOf(new Formatter().format(STORE_ORDER_BY_ID_ENDPOINT, orderId)))
-            .then()
-            .statusCode(HttpStatus.SC_NOT_FOUND)
-            .spec(buildRes())
-            .log()
-            .body()
-            .extract()
-            .response()
-            .as(ResponseInfo.class);
+                .when()
+                .get(String.valueOf(new Formatter().format(STORE_ORDER_BY_ID_ENDPOINT, orderId)))
+                .then()
+                .statusCode(HttpStatus.SC_NOT_FOUND)
+                .spec(buildRes())
+                .log()
+                .body()
+                .extract()
+                .response()
+                .as(ResponseInfo.class);
     }
 
     @Step("Sending DELETE request for non existent order to /store/order by ID {0}")
     public static ResponseInfo deleteNonExistingOrderById(long orderId) {
         LOGGER.debug("sending request");
         RequestSpecification res = RestAssured.given()
-            .spec(buildReq());
+                .spec(buildReq());
 
         LOGGER.debug("expecting response");
         return res
-            .when()
-            .delete(String.valueOf(new Formatter().format(STORE_ORDER_BY_ID_ENDPOINT, orderId)))
-            .then()
-            .statusCode(HttpStatus.SC_NOT_FOUND)
-            .spec(buildUncheckedRes())
-            .log()
-            .body()
-            .extract()
-            .response()
-            .as(ResponseInfo.class);
+                .when()
+                .delete(String.valueOf(new Formatter().format(STORE_ORDER_BY_ID_ENDPOINT, orderId)))
+                .then()
+                .statusCode(HttpStatus.SC_NOT_FOUND)
+                .spec(buildUncheckedRes())
+                .log()
+                .body()
+                .extract()
+                .response()
+                .as(ResponseInfo.class);
+    }
+
+    private static Response verifyResponse(RequestSpecification reqSpec, ERequestType requestType, String endpoint, long orderId) {
+        int actualStatusCode;
+        int i = 0;
+        String orderIdStr;
+
+        if (orderId == 0) {
+            orderIdStr = null;
+        } else {
+            orderIdStr = String.valueOf(orderId);
+        }
+
+        Response response = null;
+        do {
+            if (i == THIRTY_SECONDS) {
```

Comment:
I don't think there is need to declare THIRTY_SECONDS as class variable, private merhod variable will do just fine